cmd/lambda/leases/main_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/aws/aws-lambda-go/events"
+)
+
+func TestMain(m *testing.M) {
+	os.Setenv("LEASE_ADDED_TOPIC", "mock-lease-added-topic")
```

Comment:
Do we need to do this?  Can we just use the defaults?

Reply:
fixed